### PR TITLE
Feature/sprind 25 jwk thumbprint handling

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: packages/key-manager/src/agent/SphereonKeyManager.ts [hunk 1]
```diff
@@ -58,4 +59,24 @@ export class SphereonKeyManager extends VeramoKeyManager {
   async keyManagerListKeys(): Promise<ManagedKeyInfo[]> {
     return this.localStore.list({}) // FIXME there are no args it seems
   }
+
+  async keyManagerGet({ kid }: IKeyManagerGetArgs): Promise<IKey> {
+    try {
+      const key = await this.localStore.get({ kid })
+      return key
+    } catch (e) {
+      const keys: ManagedKeyInfo[] = await this.keyManagerListKeys()
```
Comment (nklomp) [inline packages/key-manager/src/agent/SphereonKeyManager.ts:68]: Hmzz, please create a ticket that we need to do this in a nicer way in the future.

File: packages/kms-musap-rn/src/MusapKeyManagerSystem.ts [hunk 2]
```diff
@@ -157,20 +154,22 @@ export class MusapKeyManagementSystem extends AbstractKeyManagementSystem {
...
-      publicKeyHex: publicKeyHex,
-      meta: {
-        ...args,
-      },
+      publicKeyHex,
+      meta: metadata,
     }
+
+    const jwkThumbprint = calculateJwkThumbprintForKey({ key: keyInfo as ManagedKeyInfo })
+    keyInfo.meta!.jwkThumbprint = jwkThumbprint
```
Comment (nklomp) [inline packages/kms-musap-rn/src/MusapKeyManagerSystem.ts:172]: keyInfo.meta = {...keyInfo.meta, jwkThumbprint }

**Review by nklomp — CHANGES_REQUESTED** [review]
Please create a ticket and small improvement

**Review by nklomp — CHANGES_REQUESTED** [review]
Please see small remarks, so this can be merged

File: packages/key-manager/src/agent/SphereonKeyManager.ts (hunk 1 above)
Comment (sksadjad) [reply packages/key-manager/src/agent/SphereonKeyManager.ts:68]: added the ticket and todo: https://sphereon.atlassian.net/browse/SDK-28

**Review by nklomp — APPROVED** [review]

**Pull request merged**